src/java/org/apache/cassandra/db/compaction/OperationType.java

```diff
@@ -50,15 +50,35 @@
     VIEW_BUILD("View build"),
     INDEX_SUMMARY("Index summary redistribution"),
     RELOCATE("Relocate sstables to correct disk"),
-    GARBAGE_COLLECT("Remove deleted data");
+    GARBAGE_COLLECT("Remove deleted data"),
+    DOWNGRADE_SSTABLES("Downgrade sstables"),
+    RESTORE("Restore"),
+    // operations used for sstables on remote storage
+    REMOTE_RELOAD("Remote reload", true), // reload locally sstables that already exist remotely
+    REMOTE_COMPACTION("Remote compaction", true), // no longer used, kept for backward compatibility
+    TRUNCATE_TABLE("Table truncated"),
+    DROP_TABLE("Table dropped"),
+    REMOVE_UNREADEABLE("Remote unreadable sstables"),
+    REGION_BOOTSTRAP("Region Bootstrap"),
+    REGION_DECOMMISSION("Region Decommission"),
+    REGION_REPAIR("Region Repair"),
```

Comment:
above two are also used in `SSTableAddedNotification`, do you plan to port it in a different ticket?

Reply:
good catch. I think it's best to do it together, in this ticket.
I'll look for other uses of cndb-related operation types in dse, in case they need porting too

Reply:
I created separate tickets for newly added OperationTypes, STAR-991, STAR-992, STAR-993 and STAR-994, with links to relevant CNDB tickets that need porting.